grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy

```diff
@@ -97,35 +97,34 @@ class InvoiceItem implements Serializable {
         createdBy(nullable: true)
     }
 
-    ShipmentItem getShipmentItem() {
-        return shipmentItems ? shipmentItems?.iterator()?.next() : null
+    OrderItem getOrderItem() {
+        return orderItems ? orderItems?.find { it } : null
```

Comment:
Why the switch? I can see why it might help (choose the first non-null item) but do we think it'll ever be null?

Reply:
This change was more for IDE to get rid of underscore